### ISSUE TITLE
Is there a pre-trained, plug-n-play model that we can play with?

### ISSUE THREAD

**Issue by youssefavx** [openai/deeptype#56]
I'd like to experiment with this but I don't want to train a model, I want to use an existing model (the best one available if possible)

Is there some way to do something like: 

`deeptype.run('Jaguars are running in the wild') 
`
And get an output like: 
`['Jaguar', 'wiki URL']
`
Or
`{'Jaguar': 'wiki URL'}
`
?

**Comment by ryangawei** [comment]
Agreed. I want to use a pre-trained mode for my further research. Training one seems too time-consuming.

**Comment by gloryVine** [comment]
Yeah that would be nice.

**Comment by petulla** [comment]
+1

**Comment by lbozarth** [comment]
+1

**Comment by Hadjerkhd** [comment]
Yes that would be cool !
